src/app/rundown-view/components/rundown-subheader/rundown-subheader.component.ts

```diff
@@ -0,0 +1,47 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges, SimpleChange, SimpleChanges } from '@angular/core'
+import { Rundown } from '../../../core/models/rundown'
+import { Piece } from '../../../core/models/piece'
+import { Tv2PieceMetadata } from '../../../core/models/tv2-piece'
+import { Tv2PieceType } from '../../../core/enums/tv2-piece-type'
+import { Part } from '../../../core/models/part'
+
+@Component({
+  selector: 'sofie-rundown-subheader',
+  templateUrl: './rundown-subheader.component.html',
+  styleUrls: ['./rundown-subheader.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class RundownSubheaderComponent implements OnChanges {
```

Comment:
Again, nothing in here that is header specific, so `RundownSubheaderComponent` seems to be the wrong name.

Reply:
Updated to `OnAirDetailsPanelComponent`.